Makefile

```diff
@@ -50,24 +42,34 @@ conmon:
 pause:
 	make -C $@
 
-GO_SRC =  $(shell find . -name \*.go)
-ocid: $(GO_SRC) | ${OCID_LINK}
-	$(GO) build --tags "$(BUILDTAGS)" -o $@ ./cmd/server/
+ocid:
+ifndef GOPATH
+	$(error GOPATH is not set)
+endif
+	$(GO) install \
+		-tags "$(BUILDTAGS)" \
+		$(PROJECT)/cmd/ocid
 
-ocic: $(GO_SRC) | ${OCID_LINK}
-	$(GO) build -o $@ ./cmd/client/
+ocic:
+ifndef GOPATH
+	$(error GOPATH is not set)
+endif
+	$(GO) install $(PROJECT)/cmd/ocic
 
-kpod: $(GO_SRC) | ${OCID_LINK}
-	$(GO) build -o $@ ./cmd/kpod/
+kpod:
+ifndef GOPATH
+	$(error GOPATH is not set)
+endif
+	$(GO) install $(PROJECT)/cmd/kpod
 
 ocid.conf: ocid
-	./ocid --config="" config --default > ocid.conf
+	$(GOPATH)/bin/ocid --config="" config --default > ocid.conf
 
 clean:
-	rm -f ocid.conf
-	rm -f ocic ocid
-	rm -f kpod
-	rm -f ${OCID_LINK}
+ifdef GOPATH
+	rm -rf $(GOPATH)/pkg
```

Comment:
Same for the pkg. We will have to remove $GOPATH/pkg/"$OS_$ARCH"/github.com/kubernetes-incubator/cri-o/